app/src/client/components/buildPage/FormField.tsx

```diff
@@ -60,6 +80,16 @@ export const FormField: React.FC<FormFieldProps> = ({ field, property, fieldKey,
     }
   }, [property.enum]);
 
+  const handleSelectOnchange = (selectedOption: any) => {
+    const value = selectedOption?.value || null;
+    const isAddOption = selectedOption?.isAddPropertyOption;
+
+    field.handleChange(value);
```

Comment:
Do I understand correctly, this is the place where would plug call to suspend this form, and execute new form to create new property?